src/scripts.js

```diff
@@ -46,69 +59,102 @@ function displayAllRecipes() {
 }
 
 function viewRecipeDetail(event) {
-    //GOAL: incoporate instructions and total cost into recipe details page
-    const foundRecipe = recipeRepository.recipes.find((current) => {
-        return current.id === findId(event);
+    show(saveRecipeButton)
+    hide(allRecipes);
+    hide(filterSidebar);
+    show(singleRecipe);
+    show(ingredientSidebar);
+    foundRecipe = recipeRepository.recipes.find((current) => {
+        return current.id === findId(event)
     })
-    //found.instructions is an array
-    // ${foundRecipe.instructions[i].number}. ${foundRecipe.instructions[i].instruction}
-    //Create variable and do iterator here
-    // Store in variable => insert that variable in p tag below where we hard coded!
-    // forEach to do p tag for each instrution step
-    singleRecipe.innerHTML += `
+    singleRecipe.innerHTML = `
         <img src="${foundRecipe.image}" alt="${foundRecipe.name}">
         <section class="instructions">
           <h2>${foundRecipe.name}</h2>
-          <p>${foundRecipe.instructions[0].instruction}</p>
+          <p>${foundRecipe.getInstructions()}</p>
```

Comment:
Thank you again for reminding us of this method last night! It saved us a great deal of time!